### ISSUE TITLE
Questions about FID and IS

### ISSUE THREAD

**Issue by yiiizuo** [samar-khanna/DiffusionSat#9]
I am sorry to bother you.​

 May I ask you if the real image of your paper FID is from remote sensing dataset or from ImageNet-1k? and where the weights of inception_v3 are from?

 Also, what is the source of the inception_v3 weights used to compute IS in your paper?

**Comment by samar-khanna** [comment]
The FID is calculated on the validation set of fMoW-RGB, a remote sensing dataset. I used this library to calculate FID: https://github.com/mseitzer/pytorch-fid
